src/wallet/coinselection.cpp

```diff
@@ -62,9 +62,9 @@ struct {
 
 static const size_t TOTAL_TRIES = 100000;
 
-std::optional<SelectionResult> SelectCoinsBnB(std::vector<OutputGroup>& utxo_pool, const CAmount& selection_target, const CAmount& cost_of_change)
+std::optional<SelectionResult> SelectCoinsBnB(std::vector<OutputGroup>& utxo_pool, const CAmount& selection_target, const CAmount& upper_bound)
 {
-    SelectionResult result(selection_target);
+    SelectionResult result(selection_target, /* force_no_change=*/ true);
```

Comment:
```suggestion
    SelectionResult result(selection_target, /*force_no_change=*/true);
```
Anywhere you are adding/touching named args, please use the [style from the developer docs](https://github.com/bitcoin/bitcoin/blob/master/doc/developer-notes.md#coding-style-c-named-arguments).

Reply:
fixed